qemu/tests/cfg/block_with_iommu.cfg

```diff
@@ -0,0 +1,51 @@
+- block_with_iommu:
+    type = block_with_iommu
+    only q35
+    only x86_64, i386
+    only RHEL.8
```

Comment:
conflicts with \"only Windows\"

Reply:
Thanks, have updated it, pls help review again, thaks.